visionmetrics/regression/accuracy.py

```diff
@@ -8,19 +8,17 @@ class MeanAbsoluteErrorF1Score(Metric):
     Args:
         threshold: float indicating the exclusive threshold below which the mean absolute error is considered a true positive.
```

Comment:
super irritating nit: change docstring arg to error_threshold.

Reply:
Ahh, great catch, didn't update this after renaming it in the code. Thanks!